lib/structures/Interaction.js

```diff
@@ -43,7 +43,7 @@ class Interaction extends Base {
             }
         }
 
-        this._client.emit("warn", new Error(`Unknown interaction type: ${data.type}\n${JSON.stringify(data)}`));
+        client.emit("warn", new Error(`Unknown interaction type: ${data.type}\n${JSON.stringify(data)}`));
```

Comment:
Unrelated changes should be in a separate pull request.

Reply:
I've got this in #1309